react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import logo from './logo.svg';
+import './App.css';
+
+let tasks = ["submit hyf homework", 
+            "go for a run", 
+            "watach movie"];
+
+            
+function TodoTimer(props){
+  const [count, setCount] = useState(0);
+  setTimeout(() => setCount(count + 1), 1000);
+  return <p>You have used this app for {count} seconds</p>;
+};
+
+function Todo(props){
+  
+  const [checked, updateCheck] = useState(false)
```

Comment:
```suggestion
  const [checked, setChecked] = useState(false)
```
Although it can be stupid sometimes, it is convention in React that the state and the setState functions follow the same naming and always uses the "set". So in this case it would be "checked" and "setChecked".

Reply:
ok.I will always follow. I fixed it.